content/whats_on.md

```diff
@@ -1,13 +1,15 @@
 <div class="content" markdown="1">
+
 #### Weekend-Long Speedruning Stream
-![A speedrunner and his co-commentator at the event](http://localhost:8080/dist/img/speedrunning.jpg)
+![A speedrunner and their co-commentator at the event](http://localhost:8080/dist/img/speedrunning.jpg)
 
-Two days of speedrunning in aid of [SpecialEffect](https://specialeffect.org) streamed live on the [Warwick's Awesome Speedruns and Demos](https://twitch.tv/warwickspeedrun) Twitch channel. Come and say hello in the chat!
+Two days of speedrunning in aid of [SpecialEffect](https://specialeffect.org) streamed live on the WASD [Twitch channel](https://twitch.tv/warwickspeedrun). Come and say hi in the chat!
 </div>
 
 <div class="content" markdown="1">
+
 #### Welcoming Community
 ![Spectators enjoying themselves at the event](http://localhost:8080/dist/img/community.jpg)
 
-Our events have a wonderful sense of community and we welcome everyone with open arms.
+Our events have a wonderful sense of community, and we welcome everyone with open arms!
```

Comment:
??: Why did we add this comma?

Reply:
Fixed in new commit, PyCharm inserted this one...